README.md

```diff
@@ -1,68 +1,79 @@
-# `is="castable-video"`
+# `<castable-video>`
 
 [Cast](https://developers.google.com/cast) your video element to the big screen with ease!
 
-The lightweight `CastableVideoElement` class extends the native `HTMLVideoElement` API and adds casting functionality to any video element. The API was designed to have the feel of a native browser API similar to the other screen presentation API's.
+The lightweight `CastableVideoElement` class extends the native `HTMLVideoElement` API 
+and adds casting functionality to any video element. 
+The API aims to be equivalent to the 
+[Remote Playback API](https://developer.mozilla.org/en-US/docs/Web/API/RemotePlayback)
+with a few extra element attributes specific to casting.
 
-It was primarily built for use in [Media Chrome](https://github.com/muxinc/media-chrome) but it works great with any custom video controls as you can see in the example.
+It was primarily built for use in [Media Chrome](https://github.com/muxinc/media-chrome) 
+but it works great with any custom video controls as you can see in the example.
 
 
 ```html
 <script src="https://www.gstatic.com/cv/js/sender/v1/cast_sender.js?loadCastFramework=1"></script>
-<script type="module" src="https://cdn.jsdelivr.net/npm/castable-video"></script>
+<script type="module" src="https://cdn.jsdelivr.net/npm/castable-video/+esm"></script>
 
-<video
+<castable-video
   id="castable"
-  is="castable-video"
   src="https://stream.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/high.mp4"
-></video>
+></castable-video>
+
+<button onclick="castable.play()">Play</button>
+<button onclick="castable.pause()">Pause</button>
+<button id="castBtn" hidden onclick="castable.prompt()">Cast...</button>
 
-<button onclick="window.castable.play()">Play</button>
-<button onclick="window.castable.pause()">Pause</button>
-<button onclick="window.castable.requestCast()">Request Cast</button>
-<button onclick="window.CastableVideoElement.exitCast()">Exit Cast</button>
 <script>
-  window.castable.addEventListener('castchange', function (event) {
-    console.log(event.type, event.detail);
+  castable.remote.watchAvailability((available) => {
+    castBtn.hidden = !available;
+  });
+
+  castable.remote.addEventListener('connecting', function (event) {
+    console.log(event.type);
+  });
+
+  castable.remote.addEventListener('connect', function (event) {
+    console.log(event.type);
+  });
+
+  castable.remote.addEventListener('disconnect', function (event) {
+    console.log(event.type);
   });
 </script>
 ```
 
-## API
+## Remote Playback API
+
+https://developer.mozilla.org/en-US/docs/Web/API/RemotePlayback
 
 ### Methods
 
-- `video.requestCast(options)`: open the browser casting menu. Options:
```

Comment:
yep, true this is not added back in yet.
I'll add this in before merging.
the options are set on a global level so it might be just an exported function they can call...

Reply:
added!